front/app/modules/commercial/flag_inappropriate_content/admin/components/ActivityWarningsTab.tsx

```diff
@@ -15,22 +15,29 @@ declare module 'modules/commercial/moderation/admin/containers/index' {
 
 interface Props {
   onData: (data: InsertConfigurationOptions<ITabItem>) => void;
+  activeFlagsCount: number;
 }
 
 const ActivityWarningsTab = ({
   onData,
+  activeFlagsCount,
   intl: { formatMessage },
 }: Props & InjectedIntlProps) => {
   useEffect(
     () =>
       onData({
         configuration: {
           name: 'warnings',
-          label: formatMessage(messages.warnings),
+          label: (
+            <span>
+              {`${formatMessage(messages.warnings)} (${activeFlagsCount})`}
+            </span>
+          ),
         },
         insertAfterName: 'read',
+        reinsertAfterUpdate: true,
```

Comment:
Not sure about this mechanism

Reply:
I am not sure I understand what this does?

Reply:
Well, if we're inserting a tab (in this case the ActivityWarningsTab from above) that has data (e.g. count of items shown in the tab), this data might arrive after the tab has already been inserted (pushed to the array of other tabs) and then it wouldn't be reinserted, since the code checks if the time already exists before inserting.

Is that a little bit clear? Let me know if I need to explain in more detail.

Reply:
Yes, I think I understand the problem better now, thanks! Indeed, it is a bit strange that these updates don't happen automatically and need to be specified. Is there an easy way to change the tab props without necessarily reinserting it? Then you can add the count to useEffect and it will re-render automatically. But maybe there is a limitation here I am missing?

I think this is what I meant below as well, when I said this can be handled inside the component (not really in state though so sorry if this was confusing).